examples/predictors/lstm/xprize_predictor.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
 
 import os
```

Comment:
Need to download Ox data in the case where we are using a pretrained model so did not download it in the notebook.